src/pages/guides/hybrid-plugins/getting-started/index.md

```diff
@@ -55,3 +55,54 @@ This is opposed to a standard UXP plugin’s method of accessing the file system
 let entry = await require('uxp').storage.localFileSystem.getFileForSaving("target.psd");
 document.saveAs.psd(entry);
 ```
+
+
+## Photoshop C++ SDK
+
+Photoshop offers a number of C based plugin APIs. These are documented in the [downloadable package](https://console.adobe.io/servicesandapis).
+A hybrid plugin can get access to these APIs by exposing an additional function entry point.
```

Comment:
nit again: I was suggesting leaving the colon, not the period

Reply:
Sorry I said it ambiguously. whoops!

Reply:
I went back and forth for a bit, but ultimately decided that I wasn't directly pointing to anything in the following block.